### ISSUE TITLE
Function to rename template files to package name and other goodies

### ISSUE THREAD

**Issue by bms63** [pharmaverse/admiraltemplate#80]
## Background

A person who clones the admiraltemplate repo for an extension package has a lot of housecleaning and setup to make it their own.  A function should be created that does all the chores for them.  For example, `make_extension()` would update the name of the package in the DESCRIPTION file and other files where either admiraltemplate or admiralext exixts.

## Context

This crashes our checks - maybe we could have a startup message that refers to a function that when calls updates everything for the extension pkg?  Maybe fun issue for someone to work on from community?

_Originally posted by @bms63 in https://github.com/pharmaverse/admiraltemplate/pull/79#discussion_r1623038837_

## Definition of Done

- [ ] Create function that cleans house for an extension package
  - [ ] DESCRIPTION files
  - [ ] Cleans testthat folder
  - [ ] Could place skeleton in readme similar to other admiral packages
  - [ ] Renames admiraltemplate.Rproj
  - [ ] Renames .Rbuildignore
  - [ ] Renames [admiraltemplate-package.R](https://github.com/pharmaverse/admiraltemplate/blob/main/R/admiraltemplate-package.R)
  - [ ] Others where needed (creativity is encouraged)
  - [ ] Recommendation on resources for creating hex image
-[ ] Startup Message recommending the function and then on how to remove the Startup message

@pharmaverse/admiral @pharmaverse/admiral_comm Low priority, but a fun one to work on if anyone wants to make a simple "updater" function.  It will not get a lot of use, but will make life easier for future extensions. :)

**Comment by manciniedoardo** [comment]
agreed to not implement - this process is instructive anyway when setting up an extension!
